### PR TITLE
PMM-6468 dbaas menu

### PR DESCRIPTION
**Pull request by atymchuk**
https://github.com/Percona-Lab/pmm-submodules/pull/1075#issuecomment-687086729

### CODE REVIEW

File: public/app/routes/routes.ts [hunk 1]
```diff
@@ -537,6 +537,24 @@ export function setupAngularRoutes($routeProvider: route.IRouteProvider, $locati
       controllerAs: 'ctrl',
       reloadOnSearch: false,
     })
+    .when('/graph/d/pmm-dbaas/cluster', {
```
Comment (tiagomotasantos) [inline public/app/routes/routes.ts:540]: Why the route changes and new components if we are going to send to an existing dashboard URL? 🤔

File: pkg/api/index.go [hunk 2]
```diff
@@ -358,6 +358,22 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
...
+			{Text: "Inventory", Id: "dbaas-iventory", Url: "/graph/d/pmm-dbaas/inventory", Icon: "building"},
+		}
+
+		data.NavTree = append(data.NavTree, &dtos.NavLink{
+			Text:         "SaaS",
+			SubTitle:     "Manage the SaaS cluster",
+			HideFromTabs: true,
+			Id:           "saas",
+			Icon:         "database",
+			Url:          "/graph/d/pmm-dbaas/cluster",
```
Comment (tiagomotasantos) [inline pkg/api/index.go:373]: `/graph/d/pmm-dbaas/pmm-dbaas` is the URL for the dashboard created on PMM-6469

File: pkg/api/index.go [hunk 3]
```diff
@@ -358,6 +358,22 @@ func (hs *HTTPServer) setIndexViewData(c *models.ReqContext) (*dtos.IndexViewDat
 			SortWeight:   dtos.WeightAdmin,
 			Children:     adminNavLinks,
 		})
+
+		saasNavLinks := []*dtos.NavLink{
+			{Text: "Cluster", Id: "dbaas-cluster", Url: "/graph/d/pmm-dbaas/cluster", Icon: "database"},
+			{Text: "Inventory", Id: "dbaas-iventory", Url: "/graph/d/pmm-dbaas/inventory", Icon: "building"},
```
Comment (tiagomotasantos) [inline pkg/api/index.go:364]: We only need one for now since the "Manage DB Cluster" won't exist yet.

**Review by nicolalamacchia — DISMISSED** [review]

File: public/app/routes/routes.ts (hunk 1 above)
Comment (atymchuk) [reply public/app/routes/routes.ts]: Valid, I removed the routes altogether!

File: pkg/api/index.go (hunk 3 above)
Comment (atymchuk) [reply pkg/api/index.go]: Removed that as discussed.

**Comment by atymchuk** [comment]
@tiagomotasantos yow, we shall not forget the snapshots :)

**Comment by tiagomotasantos** [comment]
This PR was overwrote by [this](https://github.com/percona-platform/grafana/pull/6) one.

**Pull request closed without merging**